architecture/README.md

```diff
@@ -12,148 +11,149 @@ types of roles. The TiDB community has the following organizations:
+    - Fixed 1 or more tasks of Hard difficulty
+- Nominated by the Tech Leads of the SIG and gained supermajority (2/3) votes from Maintainers
 
-#### Responsibilities and Privileges
+#### Privileges and responsibilities
 
-* Recommend a new Organizer Committee Member
-* The profile can be listed on the Committer page
-* Participate in offline discussions and the design process
+- Control overall code quality of projects under the SIG
+- Guide Contributors and Reviewers to contribute to the SIG continuously
+- Participate in Committer meetings regularly and have knowledge priority regarding project development
+- Participate in design discussions
+- Get the profile listed on the membership page of the corresponding SIG
 
-#### Definition of Inactive Committer and operations:
+### Maintainer
 
-If a Committer has been inactive for over three months, the Committer is an
-Inactive Committer.  Concrete inactive behaviors are as follows:
+Maintainers are the planners and designers of the TiDB project, with the authority to merge branches into the master. Maintainers come from Committers. They must demonstrate both good judgment and responsibility towards the sustained development and overall health of the TiDB project. See [TiDB Maintainers](./maintainer-list.md) for a list of TiDB Maintainers.
 
-* Absent from any community activity for over three months
-* Absent from any Committer meeting for over three months
-* Absent from technical discussions
-* No code commits (including documentation) for over three months
+#### How to become a Maintainer
 
-If a Committer has not committed any code for over a year, then the Committer
-privileges are reclaimed.
+- Demonstrated a deep and comprehensive understanding of TiDB's technical goals and directions
+- Must be Committer for at least 2 SIGs
+- Actively involved in organizing technical seminars and proposing or reviewing major technical designs
+- Can handle complex problems in the code implementation process
+- Nominated by and obtained supermajority votes from the PMC
 
-#### Committer List
+#### Privileges and responsibilities
 
-[Committer List](./committer-list.md)
+- Develop and approve technical design and decisions for TIDB projects
+- Set the technical directions and priorities for SIG projects
+- Participate in product releases and roadmap development of the project
+- Continuously contribute to the project by mentoring Reviewers and Committers
+- Ensure the overall quality of the TiDB project
 
-### Maintainer
+## TiDB User Group
 
-TBD
+The TiDB User Group (TUG) is an independent user community initiated by TiDB users. Users in the TUG community can learn cutting-edge technical knowledge, publish technical insights, get to know friends in the circle, and build TiDB projects together.
 
-#### How to become a Maintainer?
+### Leader
 
-TBD
+The Leader of the  TiDB User Group (TUG) who makes overall plans, tracks progress to strengthen the joint force and impact of the TUG.
```

Comment:
```suggestion
The Leader of the  TiDB User Group (TUG) is who makes overall plans, tracks progress to strengthen the joint force and impact of the TUG.
```

Reply:
No "is" needed. This is not a full sentence. It's quite normal to explain some concepts using this structure.

Reply:
I see~